lib/native/aws_iot.ts

```diff
@@ -78,10 +79,12 @@ export class AwsIotMqttConnectionConfigBuilder {
             clean_session: false,
             keep_alive: undefined,
             will: undefined,
-            username: `?SDK=NodeJSv2&Version=${platform.crt_version()}`,
+            //username: `?SDK=NodeJSv2&Version=${platform.crt_version()}`,
```

Comment:
should this be deleted? Or are we keeping it for reference?

Reply:
Whoops! No, it should be deleted. I just kept it there so I didn't forget what it was supposed to be. I'll remove it 👍